RELEASE-NOTES.md

```diff
@@ -4,6 +4,96 @@ section: overview
+- **About modal:** 
+  - Added type safety to default props ([#3536](https://github.com/patternfly/patternfly-react/pull/3536))
+- **Alert:** 
+  - Accessibility updates to alert for toast alerts ([#3519](https://github.com/patternfly/patternfly-react/pull/3519))
+- **Alert group:**
+    - Added alert group component ([#3531](https://github.com/patternfly/patternfly-react/pull/3531))
+- **Chip group:**
+  - Added closable chip group ([#3455](https://github.com/patternfly/patternfly-react/pull/3455))
+- **Clipboard copy:** 
+  - Set type of copy and toggle buttons to 'button' ([#3444](https://github.com/patternfly/patternfly-react/pull/3444))
+- **Data list:** 
+  - Properly assigned selectable and selected classes ([#3447](https://github.com/patternfly/patternfly-react/pull/3447))
+  - Removed hook ([#3434](https://github.com/patternfly/patternfly-react/pull/3434))
+  - Reverted React.Component back to React.FunctionComponent ([#3431](https://github.com/patternfly/patternfly-react/pull/3431))
+- **Data toolbar:** 
+  - Added and removed some modifiers to match core ([#3422](https://github.com/patternfly/patternfly-react/pull/3422))
+  - Reverted promotion of DataToolbar from experimental ([#3580](https://github.com/patternfly/patternfly-react/pull/3580))
+  - Updated example to not exclude previous filtered items. ([#3571](https://github.com/patternfly/patternfly-react/pull/3571))
+- **Divider:** 
+  - Promote Divider from experimental ([#3516](https://github.com/patternfly/patternfly-react/pull/3516))
+- **Flex layout:** Allowed string literals to be used for FlexItem modifiers ([#3544](https://github.com/patternfly/patternfly-react/pull/3544))
+- **Overflow menu:** 
+  - Promote OverflowMenu from experimental ([#3516](https://github.com/patternfly/patternfly-react/pull/3516))
+- **Page:** 
+  - Add ability to customize screen reader label ([#3433](https://github.com/patternfly/patternfly-react/pull/3433))
+- **Page header:** 
+  - Removed redundant role="banner" attribute ([#3499](https://github.com/patternfly/patternfly-react/pull/3499))
+- **Page layout:** 
+  - Fixed page layout imports for use with code sandbox.([#3461](https://github.com/patternfly/patternfly-react/pull/3461))
+- **Pagination:**
+  - Disabled navigation input if page is equal to zero ([#3472](https://github.com/patternfly/patternfly-react/pull/3472))
+- **Select:** 
+  - Add the type prop to the select toggle button ([#3495](https://github.com/patternfly/patternfly-react/pull/3495))
+  - Allowed custom option data comparison ([#3491](https://github.com/patternfly/patternfly-react/pull/3491))
+  - Updated so clear button appears with any type ahead input ([#3502](https://github.com/patternfly/patternfly-react/pull/3502))
+  - Set a unique id to the select toggle type ahead input ([#3529](https://github.com/patternfly/patternfly-react/pull/3529))
+- **Spinner** 
+ - Promote Spinner from experimental ([#3516](https://github.com/patternfly/patternfly-react/pull/3516))
```

Comment:
There's a small formatting error on spinner note. The bullet below needs to be indented.